src/nxt_http_static.c

```diff
@@ -338,17 +339,19 @@ nxt_http_static_send_ready(nxt_task_t *task, void *obj, void *data)
 
         length = shr->length + index->length;
 
-        fname = nxt_mp_nget(r->mem_pool, length + 1);
-        if (nxt_slow_path(fname == NULL)) {
+        path = nxt_str_alloc(r->mem_pool, length + 1);
```

Comment:
Mind that this switches from nxt_mp_nget() to nxt_mp_get().  I've never had the difference clear, so review that part carefully and check if that bit carefully.

```diff
$ diff -u <(grepc nxt_mp_nget .) <(grepc nxt_mp_get .)
--- /dev/fd/63	2024-01-22 12:42:19.271475515 +0100
+++ /dev/fd/62	2024-01-22 12:42:19.271475515 +0100
@@ -1,12 +1,13 @@
 ./src/nxt_mp.c:void *
-nxt_mp_nget(nxt_mp_t *mp, size_t size)
+nxt_mp_get(nxt_mp_t *mp, size_t size)
 {
     void  *p;
 
 #if !(NXT_DEBUG_MEMORY)
 
     if (size <= mp->page_size) {
-        p = nxt_mp_get_small(mp, &mp->nget_pages, size);
+        size = nxt_max(size, NXT_MAX_ALIGNMENT);
+        p = nxt_mp_get_small(mp, &mp->get_pages, size);
 
     } else {
         p = nxt_mp_alloc_large(mp, NXT_MAX_ALIGNMENT, size, 0);
@@ -18,9 +19,9 @@
 
 #endif
 
-    nxt_debug_alloc("mp %p nget(%uz): %p", mp, size, p);
+    nxt_debug_alloc("mp %p get(%uz): %p", mp, size, p);
 
     return p;
 }
-./src/nxt_mp.h:NXT_EXPORT void *nxt_mp_nget(nxt_mp_t *mp, size_t size)
+./src/nxt_mp.h:NXT_EXPORT void *nxt_mp_get(nxt_mp_t *mp, size_t size)
     NXT_MALLOC_LIKE;
```

Reply:
From what I can guess by looking at that diff, the `n` probably means "not aligned".  If so, the change is good.